docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc

```diff
@@ -0,0 +1,64 @@
+= Generic Pre-Install Checklist
+
+[abstract]
+To successfully install an OpenShift 4 cluster some pre-requisites are needed and some questions need to be answered before starting the installation. This page helps to be prepared for the cluster planning and installation.
+
+== Checklist
+
+* [ ] <<Naming and DNS>> defined?
+* [ ] <<Resource requirements>> defined?
+* [ ] <<Infrastructure>> defined?
+* [ ] <<Network requirements>> defined?
+
+== Naming and DNS
+
+DNS names are very important, changing them later is usually not possible or a huge amount of (error prone) work. Plan the following points carefully:
+
+* Base domain
+
+Also have a look at xref:explanation/dns_scheme.adoc[DNS Scheme] to get an idea how DNS records are handled.
+
+== Resource requirements
+
+The main purpose of the cluster will be to run the users applications and these applications have specific resource requirements.
+
+This needs to be taken into consideration when answering the following questions:
+
+* What kind of virtual machines will be needed to satisfy the resource requirements (vCPU / Memory) for running the user's workload?
+* How many user application Pods will be scheduled (just an estimate)?
+* How many worker nodes will be needed?
+* How much room for failure and maintenance (`n+1` redundancy) is needed?
+
+Don't go below the defined minimum requirements.
```

Comment:
Can we link here to the _defined minimum_?

Reply:
The issue here is that I don't know where to link to, as we define the minimum requirements individually per infrastructure under https://kb.vshn.ch/products/vss_ocp4.html and I didn't find a "generic" page on docs.openshift.com.

Reply:
I guess we have to figure that one out. But I am fine leave that for later.